core/src/avm2/globals/flash/events/TimerEvent.as

```diff
@@ -1,22 +1,21 @@
 package flash.events {
-	import __ruffle__.stub_method;
-	public class TimerEvent extends Event{
-		public static const TIMER:String = "timer";
-		public static const TIMER_COMPLETE:String = "timerComplete";
+    public class TimerEvent extends Event{
+        public static const TIMER:String = "timer";
+        public static const TIMER_COMPLETE:String = "timerComplete";
 
-		public function TimerEvent(type:String, bubbles:Boolean = false, cancelable:Boolean= false) {
-			super(type, bubbles, cancelable);
-		}
-		
-		override public function clone() : Event {
-			return new TimerEvent(this.type,this.bubbles,this.cancelable);
-		}
+        public function TimerEvent(type:String, bubbles:Boolean = false, cancelable:Boolean= false) {
```

Comment:
Since you also fixed some whitespace, two nits: no space in `Event{` and no space before `=`.

Reply:
Done!